src/frontend/src/handler/query.rs

```diff
@@ -156,34 +157,35 @@ pub async fn handle_query(
         }
     };
 
-    let rows_count = match stmt_type {
-        StatementType::SELECT => None,
+    let rows_count: Option<i32> = match stmt_type {
+        StatementType::SELECT | StatementType::INSERT_RETURNING | StatementType::DELETE_RETURNING | StatementType::UPDATE_RETURNING => None,
```

Comment:
Actually, in Postgres, we will get both returning rows and affected row count. We may fix this in the future.

```
postgres=# insert into t values (233) returning *;
  v  
-----
 233
(1 row)

INSERT 0 1
```

Reply:
Any ideas? 😰 cc @BowenXiao1999

Reply:
I'm also not sure, but this seems do not affect the correctness.

If the protocol allows, I think we can add the values in future. 🤔

Reply:
Agree.

Reply:
Have fixed!